ghost/portal/src/components/pages/AccountHomePage/AccountHomePage.js

```diff
@@ -38,6 +38,7 @@ export default class AccountHomePage extends React.Component {
                     onClose={() => this.context.onAction('closePopup')}
                     handleSignout={e => this.handleSignout(e)}
                     supportAddress={supportAddress}
+                    t={this.context.t}
```

Comment:
Maybe we can pull `t` out in `const {member, site} = this.context;` above and pass it in a bit cleaner here?